src/Time/Units.hs

```diff
@@ -116,6 +122,16 @@ instance KnownSymbol (ShowUnit unit) => Show (Time unit) where
                                                 n -> '/' : show n
                       in numeratorStr ++ denominatorStr ++ symbolVal (Proxy @(ShowUnit unit))
 
+instance KnownSymbol (ShowUnit unit) => Read (Time unit) where
+    readPrec = lift readP
```

Comment:
Would be good to add explicit types to `readPrec` and `readP` functions. The API of `base`'s `Read` type class is crazy and some help would be appreciated in case somebody needs to improve this instance...